src/content.js

```diff
@@ -15,13 +15,25 @@ let links = document.body.getElementsByTagName('a');
 
 // seed the content_delegate with the tabId by using the message 
 // returned from the background worker
-function addRecapInformation(msg){
-    // destructure the msg object to get the tabId
-    const { tabId } = msg;
-    
+function addRecapInformation(msg) {
+  // destructure the msg object to get the tabId
+  const { tabId } = msg;
+
+  if (PACER.isAppellateCourt(court)) {
+
+    const AppDelegate = new AppellateDelegate(tabId, court, url, links);
+
+    if (!PACER.hasPacerCookie(document.cookie)) {
+      console.info(`RECAP: Taking no actions because not logged in: ${url}`);
+      return;
+    }
+
+    AppDelegate.dispatchPageHandler();
+  } else {
+
     let content_delegate = new ContentDelegate(tabId,
       url, path, court, pacer_case_id, pacer_doc_id, links);
```

Comment:
For consistency, shouldn't we refactor this to look like the appellate example, instead having all this logic in content.js, we run a `dispatchPageHandler` function on ContentDelegate? 

Also shouldn't ContentDelegate be renamed to be district court specific (like AppellateDelegate is)?

Reply:
OK, great.